projects/npm-tools/packages/changelog-generator/src/index.js

```diff
@@ -602,6 +603,8 @@ async function main(_node, _script, ...args) {
 
 	try {
 		await go(options);
+
```

Comment:
I would add this step to the `go` function, probably, since it seems to contain all the logic of the changelog generator.

However, I see now that the `updateTags` logic has been extracted from `go`, so... :shrug: